Source/SwiftLintFramework/Rules/RuleConfigurations/ExplicitTypeInterfaceConfiguration.swift

```diff
@@ -27,27 +27,37 @@ private extension SwiftDeclarationKind {
 }
 
 public struct ExplicitTypeInterfaceConfiguration: RuleConfiguration, Equatable {
+
+    private static let variableKinds: Set<SwiftDeclarationKind> = [.varInstance,
+                                                                   .varLocal,
+                                                                   .varStatic,
+                                                                   .varClass]
+
     public var severityConfiguration = SeverityConfiguration(.warning)
 
-    var allowedKinds: Set<SwiftDeclarationKind> = [.varInstance,
-                                                   .varLocal,
-                                                   .varStatic,
-                                                   .varClass]
+    public var allowedKinds = ExplicitTypeInterfaceConfiguration.variableKinds
 
     public var consoleDescription: String {
-        return severityConfiguration.consoleDescription + ", allowed kinds: \(allowedKinds)"
+        return severityConfiguration.consoleDescription +
+        ", excluded: [\(ExplicitTypeInterfaceConfiguration.variableKinds.subtracting(allowedKinds))]"
```

Comment:
My comment here was more about how the types will be spelled out.

By just printing `SwiftDeclarationKind`, you'll get something like `source.lang.swift.decl.var.class`, but when configuring the rule you're expected to use just `class`.

Reply:
Ah I see, let me do that